public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -0,0 +1,278 @@
+import { css } from '@emotion/css';
+import { debounce } from 'lodash';
+import memoizeOne from 'memoize-one';
+import React, { useEffect } from 'react';
+
+import {
+  DataFrame,
+  ExploreLogsPanelState,
+  GrafanaTheme2,
+  Labels,
+  LogsSortOrder,
+  SplitOpen,
+  TimeRange,
+} from '@grafana/data/src';
+import { Themeable2 } from '@grafana/ui/src';
+
+import { fuzzySearch } from '../../../plugins/datasource/prometheus/querybuilder/components/metrics-modal/uFuzzy';
+import { parseLogsFrame } from '../../logs/logsFrame';
+
+import { LogsColumnSearch } from './LogsColumnSearch';
+import { LogsTable } from './LogsTable';
+import { LogsTableMultiSelect } from './LogsTableMultiSelect';
+
+interface Props extends Themeable2 {
+  logsFrames: DataFrame[];
+  width: number;
+  timeZone: string;
+  splitOpen: SplitOpen;
+  range: TimeRange;
+  logsSortOrder: LogsSortOrder;
+  panelState: ExploreLogsPanelState | undefined;
+  updatePanelState: (panelState: Partial<ExploreLogsPanelState>) => void;
+  onClickFilterLabel?: (key: string, value: string, refId?: string) => void;
+  onClickFilterOutLabel?: (key: string, value: string, refId?: string) => void;
+  datasourceType?: string;
+}
+
+export type fieldNameMeta = { percentOfLinesWithLabel: number; active: boolean | undefined };
+type fieldName = string;
+type fieldNameMetaStore = Record<fieldName, fieldNameMeta>;
+
+export const LogsTableWrap: React.FunctionComponent<Props> = (props) => {
```

Comment:
I've seen that it's not recommended to use FC or FunctionComponent, https://stackoverflow.com/questions/59988667/typescript-react-fcprops-confusion

Also doesn't seem to be used except on these files:

![imagen](https://github.com/grafana/grafana/assets/1069378/0a2dd452-ba33-48fe-9b5e-62c8d94bcf93)

Reply:
https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#how-to-declare-functional-components

Reply:
Good catch, I never cleaned up after co-pilot 🤦